<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerViewModuleBackupMonitor</class>
 <widget class="QWidget" name="CustomerViewModuleBackupMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="27" column="0" colspan="2">
    <widget class="QLabel" name="labelEnabled">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#00ff00;&quot;&gt;Backupmonitor ist aktiviert.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsThu"/>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsTue"/>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsWed"/>
   </item>
   <item row="14" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsFri"/>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsMon"/>
   </item>
   <item row="15" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsSat"/>
   </item>
   <item row="16" column="1">
    <widget class="QSpinBox" name="inputProspectiveJobsSun"/>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Ausgeschlossen von dieser ZÃ¤hlung sind Jobs mit dem Namen &amp;quot;INIT_*&amp;quot;, &amp;quot;sm_newday&amp;quot;, &amp;quot;Tagesprotokoll&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="inputInterfaceKey"/>
   </item>
   <item row="18" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="inputCustomerId"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CUSTOMER_INTERFACE_KEY:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>CUSTOMER_ID:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushGenerateNewInterfaceKey">
     <property name="text">
      <string>Generiere neuen CUSTOMER_INTERFACE_KEY</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushEnable">
       <property name="text">
        <string>Backup Monitor aktivieren</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushDisable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Backup Monitor deaktivieren</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; text-decoration: underline;&quot;&gt;Anzahl der zu erwartenden Jobs:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Dienstag:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mittwoch:</string>
     </property>
    </widget>
   </item>
   <item row="28" column="0" colspan="2">
    <widget class="QLabel" name="labelDisabled">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#ff0004;&quot;&gt;Backupmonitor ist deaktiviert.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Freitag:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Samstag:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Montag:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Donnerstag:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Sonntag:</string>
     </property>
    </widget>
   </item>
   <item row="23" column="0" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; text-decoration: underline;&quot;&gt;Client- Installation:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="24" column="0" colspan="2">
    <widget class="QTabWidget" name="tabInstallation">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Linux</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLineEdit" name="inputProxyHostname">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>http://proxyhost:3128</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="inputProxyUsername">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Username (leer = anonym):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Proxy- Hostname (oder leer):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="inputProxyPassword"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Proxy- Passwort:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushClearProxyInputs">
           <property name="text">
            <string>leeren</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;BenÃ¶tigte Pakete: php5, (php5-cli), php5-pgsql (bei 64bit), php5-zlib, php5-bz2, php5-openssl, php5-curl&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="inputClientInstallationCommand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Windows</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Download:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Installation muss mit dem Backupsyncer- MSI Paket durchgefÃ¼hrt werden.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelMsiDownload">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputProspectiveJobsMon</tabstop>
  <tabstop>inputProspectiveJobsTue</tabstop>
  <tabstop>inputProspectiveJobsWed</tabstop>
  <tabstop>inputProspectiveJobsThu</tabstop>
  <tabstop>inputProspectiveJobsFri</tabstop>
  <tabstop>inputProspectiveJobsSat</tabstop>
  <tabstop>inputProspectiveJobsSun</tabstop>
  <tabstop>inputInterfaceKey</tabstop>
  <tabstop>inputCustomerId</tabstop>
  <tabstop>pushGenerateNewInterfaceKey</tabstop>
  <tabstop>pushEnable</tabstop>
  <tabstop>pushDisable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
